schema/JF_TRAN.json

```diff
@@ -295,19 +296,18 @@
     "contribution_date": {
       "title": "CONTRIBUTION DATE",
       "description": "",
-      "type": "integer",
-      "minimum": 0,
-      "maximum": 99999999,
-      "examples": [
-        20120615
-      ],
+      "type": "string",
+      "minLength": 8,
+      "maxLength": 8,
+      "pattern": "^\\d{8}$",
```

Comment:
Can we make this match #87 so we don't accidentally overwrite the changes from that branch.
https://github.com/fecgov/fecfile-validate/blob/c9566940cd902cd2258b643cba0fabb4de3dc722/schema/JF_TRAN.json#L262

Reply:
Looks like the contribution_date was updated to the YYYY-MM-DD format so resolving this comment